src/retrace/retrace_worker.py

```diff
@@ -432,8 +429,7 @@ def start_retrace(self, custom_arch=None):
                     mockcfg.write("name=%s\n" % releaseid)
                     mockcfg.write("baseurl=file://%s/\n" % repopath)
                     mockcfg.write("failovermethod=priority\n")
-                    if version != "rawhide" and CONFIG["RequireGPGCheck"]:
-                        mockcfg.write("gpgkey=file:///usr/share/retrace-server/gpg/%s-%s\n" % (distribution, version))
+                    mockcfg.write("gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-fedora-%s-%s" % (version, arch))
```

Comment:
Hmmm, does EL8 knows about fedora keys? Can we use distribution-gpg-keys package?

Reply:
Bit of a chicken-egg situation because we need to install `distribution-gpg-keys` first, for which we need a GPG key from `distribution-gpg-keys`?

Reply:
You can install it on host.